inbox/pubsub-server-info.xml

```diff
@@ -0,0 +1,275 @@
+    <jid>guus.der.kinderen@igniterealtime.org</jid>
+  </author>
+  <revision>
+    <version>1.0.0</version>
+    <date>2023-12-19</date>
+    <initials>gdk</initials>
+    <remark>
+      <ul>
+        <li>Initial version.</li>
+      </ul>
+    </remark>
+  </revision>
+</header>
+<section1 topic="Introduction" anchor="intro">
+  <p>To facilitate discovery of information of individual domains in an XMPP-based network, this specification defines a data format to define basic information for individual XMPP domains. By leveraging &xep0060; this information can efficiently be shared with applications that compose an overview of the larger XMPP network.</p>
+</section1>
+  <section1 topic="Requirements" anchor="requirements">
+    <ul>
+      <li>Describe links between nodes in an XMPP-based network, by enumerating connections used for federation between XMPP domains.</li>
+      <li>An extensible data format, allowing additional data (such as that defined in &xep0092;) to be retrievable without requiring additional round-trips.</li>
+    </ul>
+  </section1>
+<section1 topic="Discovering Support" anchor="disco">
+  <p>Domains supporting the publication of Server Information data, as described in this document, MUST advertise the fact by announcing a &xep0030; feature of 'urn:xmpp:serverinfo:0'. This signifies that an administrative entity approved the publication of data, which is important for the opt-in mechanism described in <link url="#privacy">Privacy Considerations section</link> of this document.</p>
+  <p>The pub-sub service address and node in which Server Information data is advertised SHOULD be specified using a &xep0128;. These pub-sub coordinates MUST be scoped using a FORM_TYPE of "http://jabber.org/network/serverinfo" (as already specified in XEP-0128) and data form fields registered for this purpose as defined in the <link url="#registrar">XMPP Registrar Considerations section</link> of this document.</p>
+  <p>When the 'urn:xmpp:serverinfo:0' feature but no corresponding Service Discovery Extension is advertised, the node that is used will be a first-level leaf node using the name 'serverinfo' on the first pub-sub service advertised through service discovery.</p>
+  <example caption="Service Discovery information request"><![CDATA[
+<iq type='get'
+    from='francisco@denmark.lit/barracks'
+    to='shakespeare.lit'
+    id='disco1'>
+  <query xmlns='http://jabber.org/protocol/disco#info'/>
+</iq>]]></example>
+  <example caption="Service Discovery information response"><![CDATA[
+<iq type='result'
+  from='shakespeare.lit'
+  to='francisco@denmark.lit/barracks'
+  id='disco1'>
+  <query xmlns='http://jabber.org/protocol/disco#info'>
+    ...
+    <feature var='urn:xmpp:serverinfo:0'/>
+    ...
+    <x xmlns='jabber:x:data' type='result'>
+      <field var='FORM_TYPE' type='hidden'>
+        <value>http://jabber.org/network/serverinfo</value>
+      </field>
+      <field var='serverinfo-pubsub-service'>
```

Comment:
[XEP-0060 specifies an URI format for PubSub](https://xmpp.org/extensions/xep-0060.html#impl-uri) which should be used here, instead of splitting the information into two fields.